examples/2d/elixir_euler_blob_split_shockcapturing_amr.jl

```diff
@@ -0,0 +1,79 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+gamma = 5/3
+equations = CompressibleEulerEquations2D(gamma)
+
+initial_conditions = initial_conditions_blob
+
+surface_flux = flux_lax_friedrichs
+volume_flux  = flux_chandrashekar
+basis = LobattoLegendreBasis(4)
+
+indicator_sc = IndicatorHennemannGassner(equations, basis,
+                                         alpha_max=0.4,
+                                         alpha_min=0.0001,
+                                         alpha_smooth=true,
+                                         variable=pressure)
+
+volume_integral = VolumeIntegralShockCapturingHG(indicator_sc;
+                                                 volume_flux_dg=volume_flux,
+                                                 volume_flux_fv=surface_flux)
+
+solver = DGSEM(basis, surface_flux, volume_integral)
+
+coordinates_min = (-20, -20)
+coordinates_max = ( 20,  20)
+
+mesh = TreeMesh(coordinates_min, coordinates_max,
```

Comment:
Just a thought: Should we support specifying a domain by giving a center and the extent? Or would that just be confusing to users/not add value but only code?

Reply:
just reading this I am not sure what you mean. Would this be for vortex type problems? Like centered at (0,0) and then set something like 2 units in both x/y directions because we do not have rectangular domain capabilities. I think this is a very weird way to generate the mesh

Reply:
I meant in general. But your response already indicates clearly that this was not a good suggestion from my side, so I'll close this here.